libpod/container_api.go

```diff
@@ -103,6 +103,15 @@ func (c *Container) Init() (err error) {
 		return errors.Wrapf(err, "unable to generate hostname file for container")
 	}
 
+	imageVolMounts := []spec.Mount{}
+	if c.config.ImageVolumes {
+		imageVolMounts, err = c.addImageVolumes(c.config.Spec.Mounts)
```

Comment:
Can you make this work on g, the config generator, and not `c.config.Spec.Mounts`? The generator creates our running config, `c.config.Spec.Mounts` should never change after we create the container

Reply:
okay, fixed.